backend/pom.xml

```diff
@@ -182,6 +190,12 @@
       <artifactId>mssqlserver</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>net.java.dev.jna</groupId>
+      <artifactId>jna</artifactId>
```

Comment:
Out of curiosity, what is this used for?

Reply:
The JNA dependency was needed for my local setup, isn't actually needed in the UI code. So I will remove it.